### PR TITLE
Moved the admin org config endpoint to /admin/orgs

### PR DESCRIPTION
**Pull request by ryanpbrewster**
And made it return a chunked response so it won't timeout when we call it.

Testing a chunked response turned out to be more complex than I thought. Kind
of neat the way that Play handles them.

@stkem @camhashemi

### CODE REVIEW

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/admin/AdminOrganizationControllerTest.scala [hunk 1]
```diff
@@ -26,5 +32,106 @@ class AdminOrganizationControllerTest extends Specification with ShoeboxTestInje
...
+              //     XYZ
+              //     <actual stuff>
+              // The XYZ is some sort of indicator about the size of the chunk coming down
+              val s = new String(c).dropWhile(!_.isWhitespace)
+              Json.parse(s)
+            }
+            JsArray(payloads)
+          }
+        }
+      }
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/admin/AdminOrganizationControllerTest.scala:53]: @stkem here's where I extract the body from the chunked response.

Comment (stkem) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/admin/AdminOrganizationControllerTest.scala:53]: Nice!

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/admin/AdminOrganizationControllerTest.scala [hunk 2]
```diff
@@ -26,5 +32,106 @@ class AdminOrganizationControllerTest extends Specification with ShoeboxTestInje
...
+
+          db.readOnlyMaster { implicit session =>
+            orgs.foreach { org =>
+              orgConfigRepo.getByOrgId(org.id.get).settings === plan.defaultSettings
+            }
+          }
+          1 === 1
+        }
+      }
+      "work for both adding and removing features" in {
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/admin/AdminOrganizationControllerTest.scala:92]: thanks :+1:

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/admin/AdminOrganizationControllerTest.scala:92]: :ok_hand: Thanks for always writing really robust tests. Sometimes it catches me by surprise.

**Pull request merged**